core/src/components/checkbox/checkbox.scss

```diff
@@ -148,46 +148,53 @@ input {
   opacity: 0;
 }
 
-// Justify Content
-// ---------------------------------------------
+// Checkbox Bottom Content
+// ----------------------------------------------------------------
+
+.checkbox-bottom {
+  @include padding(4px, null, null, null);
+
+  display: flex;
 
-:host(.checkbox-justify-space-between) .checkbox-wrapper {
   justify-content: space-between;
-}
 
-:host(.checkbox-justify-start) .checkbox-wrapper {
-  justify-content: start;
+  font-size: dynamic-font(12px);
```

Comment:
Same as above. This should be separated and on Ionic theme use the 
body.md.regular token.

Reply:
https://github.com/ionic-team/ionic-framework/pull/30140#discussion_r1969963015